rpcs3/Emu/Cell/Modules/sceNp2.h

```diff
@@ -1413,7 +1413,7 @@ struct SceNpMatching2RoomMessageInfo
 	u8 padding[2];
 	vm::bptr<SceNpMatching2RoomMessageDestination> dst;
 	vm::bptr<SceNpUserInfo2> srcMember;
-	vm::bcptr<void> msg;
+	vm::bptr<void> msg;
```

Comment:
? seems to be correct ?

Reply:
The constness of that part of the struct only matters from the game perspective and it breaks my memory allocator allocate function.